.github/workflows/releaseBuild.yaml

```diff
@@ -0,0 +1,127 @@
+# ReleaseBuild Workflow
+#
+# This GitHub Actions workflow performs the publication process for a FHIR IG Release.
+#
+# Based on these workflows:
+# https://github.com/medigree/base/blob/main/.github/workflows/ghbuild.yml
+# https://github.com/WorldHealthOrganization/smart-html/blob/main/.github/workflows/release.yml
+
+name: "ReleaseBuild"
+
+# Controls when the action will run.
+on:
+  workflow_call: # Reusable by other workflows.
+  # Triggers the workflow on push or pull request events.
+  push:
+    branches: 
+      # Only triggers on pushes on these branches.
+      - "master"
```

Comment:
Just for my understanding: when testing additions to the IG, is it still possible to have a branch built, or does it require a pull request?

Reply:
It requires a pull request, but you can still updated the branch and the build on the PR will be rebuilt.